### ISSUE TITLE
[Bug]: 'Continue' button on Send Bitcoin screen doesn't work when animations are disabled in android

### ISSUE THREAD

**Issue by catch-21** [synonymdev/bitkit-react-native#2056]
### Describe the bug

If I disable animations in android accessibility system settings then the 'Continue' button on the "Send Bitcoin" screen does not work.

### Reproduce

1. Disable animations in android's accessiblity systen settings
2. Launch Bitkit
3. Send
4. Enter Manually
5. Input valid LN address
6. Tap 'Send'
7. Observe nothing happens

### Screenshots / Recording

![Screenshot 2024-07-11 at 16 27 29](https://github.com/synonymdev/bitkit/assets/74595920/01e29a41-028d-486e-8d61-020f8099ab6b)

https://github.com/synonymdev/bitkit/assets/74595920/b5e6f8bc-f7f8-4447-a57d-5a9121e54400

### Operating system

Android 13 TKQ1.221114.001

### Bitkit version

v128

**Comment by limpbrains** [comment]
Continue button is not working because the LN address is incorrect, and WOS returns an error. It exhibits the same behavior as any other wrong address, such as on-chain or LN invoice.

**Comment by catch-21** [comment]
For sure, the address is valid because I am able to use it in Bitkit with android animations enabled (defeault state).
Perhaps this button pressing is not directly related to animations. Could it be related to the following sheet that is being blocked?

**Comment by pwltr** [comment]
@catch-21 Could you retest this on latest master? I think this one was actually fixed with https://github.com/synonymdev/bitkit/pull/2050. It includes a fix for reduced motion setting messing with bottom sheets on Android.

**Comment by catch-21** [comment]
It does indeed appear to be resolved in latest, tested on d9e9e3df4d3f050776af76b0c6025d81063e8233. Closing.
